### PR TITLE
Add event types to dependency config

### PR DESCRIPTION
**Pull request by juliotrigo**
* Rename `GENERIC_EVENT_TYPE` constant
* Add event types to dependency config

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,6 +69,12 @@ Enable auto capture event logs in your nameko configuration file:
     EVENTLOG_DISPATCHER:
       auto_capture: true
       entrypoints_to_exclude: []
+      event_types:
+        generic: log_event
+        entrypoint_fired: entrypoint_fired
```
Comment (heindsight) [inline README.rst]: I think the config structure might get a bit confusing here. Part of the problem is probably that we are overloading the term `event_type`. Maybe we should rename the `event_type` in the payload to just `event` and change the config to something like:
```
EVENTLOG_DISPATCHER:
  auto_capture: true
  entrypoints_to_exclude: []
  event_type: log_event
  entrypoint_fired_event: entrypoint_fired
```

File: nameko_eventlog_dispatcher/eventlog_dispatcher.py [hunk 2]
```diff
@@ -28,20 +28,31 @@ class EventLogDispatcher(EventDispatcher):
     ENTRYPOINT_FIRED = 'entrypoint_fired'
     """Event type used by events triggered when entrypoints are fired."""
 
-    EVENT_TYPE = 'log_event'
+    GENERIC_EVENT_TYPE = 'log_event'
```
Comment (heindsight) [inline nameko_eventlog_dispatcher/eventlog_dispatcher.py]: I think it's better to keep this as `EVENT_TYPE`.

Comment (heindsight) [reply nameko_eventlog_dispatcher/eventlog_dispatcher.py]: Or maybe we can make this `NAMEKO_EVENT_TYPE` (and change the config key to `nameko_event_type`).

File: README.rst (hunk 1 above)
Comment (juliotrigo) [reply README.rst]: Thanks for the suggestion. I see how these different event types could be confusing.
It seems to me that the best solution for now will be to add the ability to provide an alternative Nameko event type in the config, which will be the event type to handle by the entrypoints.

```
EVENTLOG_DISPATCHER:
  auto_capture: true
  entrypoints_to_exclude: []
  event_type: log_event
```
 
The `entrypoint_fired` event type is still an event type, but not a Nameko event type and just an application level event type (for the services using the dependency) that we want to log and goes into the event metadata. This is not something that will affect how services interact with each other and will not be configurable in the config. For now, if we ever have the need to change it, we can always subclass:

```python
from nameko_eventlog_dispatcher import EventLogDispatcher

class MyEventLogDispatcher(EventLogDispatcher):
    ENTRYPOINT_FIRED = 'my_entrypoint_fired'
```

**Review by heindsight — APPROVED** [review]
:+1:

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,5 @@
+include CONTRIBUTORS.txt
```
Comment (gianchub) [inline MANIFEST.in:1]: You want to include contributors and readme in the dist?

**Review by gianchub — APPROVED** [review]

File: MANIFEST.in (hunk 3 above)
Comment (juliotrigo) [reply MANIFEST.in:1]: I've checked some open source libraries and they're normally included. For instance, Django also does it: https://github.com/django/django/blob/master/MANIFEST.in

The README file is automatically included and I think it's also a good idea to distribute the list of contributors and the LICENSE as part of the dist.

Comment (gianchub) [reply MANIFEST.in:1]: Cool

**Pull request merged**